hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -230,6 +237,16 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
         {deletePendingSummarytotalDeletedKeys !== undefined ? deletePendingSummarytotalDeletedKeys: '0'}  <span className='ant-card-meta-description meta'>Total Pending Delete Keys</span>
       </div>
   );
+    const scmServiceData = (
+        <div>
+          Service Id: <span className="ant-card-meta-description meta" style={{fontSize: "17px"}}>{scmServiceId}</span><br />
+        </div>
+    );
+    const omServiceData = (
+        <div>
+          Service Id: <span className="ant-card-meta-description meta" style={{fontSize: "17px"}}>{omServiceId}</span>
+        </div>
+    );
```

Comment:
I think since it's separated, we can follow the format of the other overview card (e.g. "Datanodes", "Volumes", etc). We can remove the "Service Id:" since I find it a bit odd that the it is larger than the actual Service ID (e.g. "omservice" is smaller than the "Service Id:" description). 

<img width="1619" alt="image" src="https://github.com/apache/ozone/assets/36403683/e3271e81-77fc-461b-8a6e-28e77ac40708">

I think this can be achieved by just removing these lines and take directly from the `omServiceId` and `scmServiceId` directly.